docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4385,29 +4385,30 @@ them, and/or process updates in parallel.  Updates across multiple concurrent
 `WriteRequest`s can also be processed interleaved and/or in parallel.
 However, **the processing of updates must be strictly serializable**.  That
 is, given a history $S$ of `WriteRequest`s including the responses to those
-requests, there must exist an order $L$ for all updates in $S$, such that:
+requests, there must exist an order $L$ for all requests in $S$, such that:
 
-1. For two updates $u_1$ and $u_2$, if the write request containing $u_1$
-   completed before the write request of $u_2$ was sent, then $u_1$ must appear
-   before $u_2$ in $L$.
-2. Executing all updates in $L$ sequentially must yield the same response for
-   every update as in $S$.
+1. For two `WriteRequest`s $r_1$ and $r_2$, if $r_1$ completed before $r_2$ was
+   sent, then $r_1$ must appear before $r_2$ in $L$.
+2. Executing all requests in $L$ sequentially must yield the same response for
+   every request as in $S$, &ie; the status reported by the server for every
```

Comment:
Another problem with doing this at the request level is that "Executing all requests in $L$ sequentially must yield the same response for every request as in $S$" is not really defined, what does it mean to execute a request sequentially in terms of the updates within a request?